server/services/store/store.go

```diff
@@ -62,4 +63,8 @@ type Store interface {
 	HasWorkspaceAccess(userID string, workspaceID string) (bool, error)
 	GetWorkspaceCount() (int64, error)
 	GetUserWorkspaces(userID string) ([]model.UserWorkspace, error)
+
+	GetBlocksWithSameID() ([]model.Block, error)
```

Comment:
I think this methods no longer need to be exposed.

Reply:
Addressed in 7f1da7c